lib/cli/src/commands/init.rs

```diff
@@ -0,0 +1,605 @@
+use anyhow::Context;
+use cargo_metadata::{CargoOpt, MetadataCommand};
+use clap::Parser;
+use std::collections::HashMap;
+use std::path::Path;
+use std::path::PathBuf;
+
+static NOTE: &str =
+    "# See more keys and definitions at https://docs.wasmer.io/ecosystem/wapm/manifest";
+
+const NEWLINE: &str = if cfg!(windows) { "\r\n" } else { "\n" };
+
+/// CLI args for the `wasmer init` command
+#[derive(Debug, Parser)]
+pub struct Init {
+    /// Initialize wapm.toml for a library package
+    #[clap(long, group = "crate-type")]
+    pub lib: bool,
+    /// Initialize wapm.toml for a binary package
+    #[clap(long, group = "crate-type")]
+    pub bin: bool,
+    /// Initialize an empty wapm.toml
+    #[clap(long, group = "crate-type")]
+    pub empty: bool,
+    /// Force overwriting the wapm.toml, even if it already exists
+    #[clap(long)]
+    pub overwrite: bool,
+    /// Don't display debug output
+    #[clap(long)]
+    pub quiet: bool,
+    /// Ignore the existence of cargo wapm / cargo wasmer
+    #[clap(long)]
+    pub no_cargo_wapm: bool,
+    /// Namespace to init with, default = current logged in user or _
+    #[clap(long)]
+    pub namespace: Option<String>,
```

Comment:
We shouldn't ask for the namespace in general.
There is one package name: `python/python` is a package name. The frontend should not care about separating the package name from the namespace. The frontend (cli) should only validate that the package name is valid as per regex. That's it

Reply:
okay, changed it to a `package-name` field